### ISSUE TITLE
Textmate will not save changes to remote volumne (Google Drive)

### ISSUE THREAD

**Issue by epurdom** [textmate/bugs#38]
Hello,

I am on Mac OS Catalina (10.15.7), running TextMate version 2.0.23. I have previously worked with Google's Backup and Sync with my documents and have edited and saved the results in TextMate without a problem. Google has just forced a switched to a new Google Drive app. I don't know the details of how these differ, other than my Google drive folder is now a mounted external volume under `/Volumes`, where previously it was a folder in my user directory. 

The problem is that now when I work with documents on `/Volumes/Google Drive`, Textmate will not save the file . I get a message, "The document “XYZ” could not be saved. Setting extended attributes". 

I would note that if I make a *new file* within Textmate and save it on `/Volumes/Google Drive`, I can save the new document with no problem (and it shows up synced in the cloud etc). It is only a problem if I try to save edits to an existing file. Moreover other text editors do NOT have this problem (e.g. TextEdit in Mac) so it appears to be something different in how Textmate is interfacing with the remote volume. 

In summary:
1) Make new document test.txt in Textmate and save on `/Volumes/Google Drive/My Drive` [Works]
2) Edit existing file test.txt in Textmate and save it [Doesn't work]
3) Edit existing file test.txt in TextEdit and save it [Works]

One difference with the new new Google drive app is that by default it does not download the files to the desktop, but only streams what you need. Again, I don't know the details, but I do know that there is an option to have an individual folder be downloaded to the desktop for offline usage (available by right-clicking the file), and I see the same behavior either way. 

Thanks!
Elizabeth

**Comment by infininight** [comment]
We are using extended attributes for storing things like the caret position, the file system is reporting that it supports this but doesn't. Is a little odd that it works for an initial save. You can disable this feature with the following in a `.tm_properties` file saved in your home directory:

```
[ '/Volumes/Google Drive/**' ]
disableExtendedAttributes = true
```
